straxen/plugins/high_energy_processing.py

```diff
@@ -0,0 +1,155 @@
+import strax
+import straxen
+import numpy as np
+
+export, __all__ = strax.exporter()
+
+# These are also needed in peaklets, since hitfinding is repeated
+HITFINDER_OPTIONS_he = tuple([
+    strax.Option(
+        'hit_min_amplitude_he',track=False,
+        default=tuple(np.ones(800, dtype=np.int16) * 15),
+        help='Minimum hit amplitude in ADC counts above baseline. '
+             'See straxen.hit_min_amplitude for options.'
+    )])
+
+
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_tpc_pmts_he',track=False,default=800
+        ),
+    strax.Option(
+        'parent_version',track=True,default=straxen.PulseProcessing.__version__
+        ),
+    *HITFINDER_OPTIONS_he)
+class PulseProcessingHe(straxen.PulseProcessing):
+    __version__ = '0.0.1'
+    rechunk_on_save = False
+    depends_on = 'raw_records_he'
+    provides = 'records_he'
+    data_kind = 'records_he'
+
+    def infer_dtype(self):
+        dtype = strax.record_dtype(strax.DEFAULT_RECORD_LENGTH)
+        return dtype
+
+    def setup(self):
+        self.hev_enabled=False
+        self.config['n_tpc_pmts'] = self.config['n_tpc_pmts_he']
+        self.config['hit_min_amplitude'] = self.config['hit_min_amplitude_he']
+
+    def compute(self, raw_records_he, start, end):
+        result = super().compute(raw_records_he, start,end)
+        return result['records']
+
+    
+@export
+@strax.takes_config(
+    strax.Option(
+        'n_tpc_pmts_he',track=False,default=800
+        ),
+    strax.Option(
+        'he_channels_offset',track=False,default=500
+        ),
+    strax.Option(
+        'amplification',track=False,default=20
+        ),
+    strax.Option(
+        'parent_version',track=True,default=straxen.Peaklets.__version__
+        ),
+    *HITFINDER_OPTIONS_he
+)
```

Comment:
Like Joran already commented it, could you also add the other options for peakelts here. You can keep the parameters as they are for the normal channels. But I think in the future we will need those options. So better to add them right in the beginning.

Reply:
Same for the other plugins, I just checked their options and I think at some point they will be all needed.